app/src/main/java/com/github/codetanzania/util/ImageUtils.java

```diff
@@ -14,6 +14,9 @@
 
 public class ImageUtils {
 
```

Comment:
Please add smarter image compression for larger file attachments, as documented here: https://developer.android.com/topic/performance/graphics/load-bitmap.html